crawl-ref/source/aptitudes.h

```diff
@@ -276,6 +276,45 @@ static const species_skill_aptitude species_skill_aptitudes[] =
     APT(SP_KOBOLD,          SK_INVOCATIONS,     1),
     APT(SP_KOBOLD,          SK_EVOCATIONS,      2),
 
+    // SP_MOUNTAIN_DWARF
+    APT(SP_MOUNTAIN_DWARF,  SK_FIGHTING,        2),
+    APT(SP_MOUNTAIN_DWARF,  SK_SHORT_BLADES,   -2),
+    APT(SP_MOUNTAIN_DWARF,  SK_LONG_BLADES,    -2),
+    APT(SP_MOUNTAIN_DWARF,  SK_AXES,            2),
+    APT(SP_MOUNTAIN_DWARF,  SK_MACES_FLAILS,    4),
+    APT(SP_MOUNTAIN_DWARF,  SK_POLEARMS,       -1),
+    APT(SP_MOUNTAIN_DWARF,  SK_STAVES,          0),
+    APT(SP_MOUNTAIN_DWARF,  SK_SLINGS,          0),
+    APT(SP_MOUNTAIN_DWARF,  SK_BOWS,           -2),
+    APT(SP_MOUNTAIN_DWARF,  SK_CROSSBOWS,       2),
+    APT(SP_MOUNTAIN_DWARF,  SK_THROWING,        0),
+    APT(SP_MOUNTAIN_DWARF,  SK_ARMOUR,          2),
+    APT(SP_MOUNTAIN_DWARF,  SK_DODGING,        -2),
+    APT(SP_MOUNTAIN_DWARF,  SK_STEALTH,        -2),
+    #if TAG_MAJOR_VERSION == 34
+    APT(SP_MOUNTAIN_DWARF,  SK_STABBING,      UNUSABLE_SKILL),
+    #endif
+    APT(SP_MOUNTAIN_DWARF,  SK_SHIELDS,         0),
+    #if TAG_MAJOR_VERSION == 34
+    APT(SP_MOUNTAIN_DWARF,  SK_TRAPS,         UNUSABLE_SKILL),
+    #endif
+    APT(SP_MOUNTAIN_DWARF,  SK_UNARMED_COMBAT,  1),
+    APT(SP_MOUNTAIN_DWARF,  SK_SPELLCASTING,   -1),
+    APT(SP_MOUNTAIN_DWARF,  SK_CONJURATIONS,   -4),
+    APT(SP_MOUNTAIN_DWARF,  SK_HEXES,          -2),
```

Comment:
-4 is worse than the worst-apt races (Sp, Vp, Ha) for conjurations - are you sure you want that? It would make MDCj almost unplayably bad.

Reply:
Yes, it compensates for +2 fire and no armor penalty. I'm trying to encourage hybrid characters instead of direct damage spells.